arcgis-ios-sdk-samples/Utility Network/Perform valve isolation trace/PerformValveIsolationTrace.storyboard

```diff
@@ -0,0 +1,116 @@
+                                        <rect key="frame" x="0.0" y="0.0" width="414" height="20.5"/>
+                                        <fontDescription key="fontDescription" style="UICTFontTextStyleBody"/>
+                                        <nil key="textColor"/>
+                                        <nil key="highlightedColor"/>
+                                    </label>
+                                </subviews>
+                                <constraints>
+                                    <constraint firstItem="JKh-re-nRw" firstAttribute="trailing" secondItem="C4m-2x-Lci" secondAttribute="trailing" id="9qR-Tv-aPH"/>
+                                    <constraint firstAttribute="bottom" secondItem="JKh-re-nRw" secondAttribute="bottom" id="Obm-69-Fq7"/>
+                                    <constraint firstItem="JKh-re-nRw" firstAttribute="top" secondItem="C4m-2x-Lci" secondAttribute="top" id="UDd-Ub-HfA"/>
+                                    <constraint firstItem="JKh-re-nRw" firstAttribute="leading" secondItem="C4m-2x-Lci" secondAttribute="leading" id="Zip-g4-0yK"/>
+                                </constraints>
+                                <viewLayoutGuide key="safeArea" id="C4m-2x-Lci"/>
+                            </view>
+                            <toolbar opaque="NO" clearsContextBeforeDrawing="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="mcq-w1-aYB">
+                                <rect key="frame" x="0.0" y="818" width="414" height="44"/>
+                                <items>
+                                    <barButtonItem style="plain" id="w6l-Ji-WPD">
+                                        <view key="customView" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="5Ke-ZQ-uR3">
+                                            <rect key="frame" x="20" y="22" width="130" height="0.0"/>
+                                            <subviews>
+                                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Isolated  Features" textAlignment="natural" lineBreakMode="tailTruncation" numberOfLines="3" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="lHG-2p-x4N">
+                                                    <rect key="frame" x="0.0" y="-17" width="70" height="34"/>
+                                                    <constraints>
+                                                        <constraint firstAttribute="width" relation="greaterThanOrEqual" constant="70" id="Yay-jv-ubm"/>
```

Comment:
The label shouldn't need a width constraint. The width should be determined automatically based on the label's text.